services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -387,8 +405,21 @@ async def __load_projects(
                 _check_project_permissions(row, user_id, user_groups, "read")
             except ProjectInvalidRightsError:
                 continue
+            try:
+                import concurrent.futures
+
+                with concurrent.futures.ThreadPoolExecutor() as pool:
+                    await asyncio.get_event_loop().run_in_executor(
+                        pool, ProjectAtDB.from_orm, row
+                    )
+                # ProjectAtDB.from_orm(row)
```

Comment:
MNIOR: remove commented 

Q: does it really make a difference making it async? Did you measure?

Reply:
I'm still getting "some" 503 when the system more so I took no chance.